gmd.py

```
@@ -8,22 +8,103 @@
 from pathlib import Path
 from typing import List, Optional
```

Ruff (0.15.9)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)